### PR TITLE
Update nl web translation for 3.9.0

### LINKED ISSUE #778: Update nl web translation for 3.9.0
Hi @henkdegroot,

We are getting ready for the 3.9.0 release and have created the [next-release](https://github.com/jamulussoftware/jamuluswebsite/tree/next-release) branch ([full diff](https://github.com/jamulussoftware/jamuluswebsite/compare/release..next-release)).

We would be happy if you updated the translations for **nl** by **2022-07-22**.

Please

- Start your work in your fork on a branch based on jamuluswebsite's `next-release` branch.
- Update the language-specific files using your favorite editor (or directly on Github),
- New/changed images are listed at the end of this issue. [Generate new URLs](https://github.com/jamulussoftware/jamuluswebsite/tree/release#adding-screenshots) for your image `.inc` files.
- Commit and push your changes to your fork,
- Open a Pull Request with your translations to the **next-release** branch with the subject `Update nl web translation for 3.9.0`,
- Link your PR(s) to this issue by including `Fixes #778` in the PR content.

Suggested commit and PR message:
```
Update nl web translation for 3.9.0

Fixes #778
```

Feel free to use this Issue to discuss anything you need prior to making any PR (and in your native language if needed).

Further documentation can be found in [TRANSLATING.md](https://github.com/jamulussoftware/jamuluswebsite/blob/next-release/README.md#translating).

Thanks for contributing to Jamulus!

---

**New/Changed screenshots**:

Please [replace with new URLs](https://github.com/jamulussoftware/jamuluswebsite/tree/release#adding-screenshots) for your screenshots here, and use those URLs in your `.inc` files.

settings-profile

server-window-setup

server-window-options

### PR DESCRIPTION
**Pull request by henkdegroot**
**Short description of changes**

Update nl web translation for 3.9.0

**Context: Fixes an issue? Related issues**

Fixes #778 

**Status of this Pull Request**

Completed

**What is missing until this pull request can be merged?**

Nothing

**Does this need translation?**
No

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

File: _translator-files/po/nl/Include-Server-Commands.po [hunk 1]
```diff
@@ -6,47 +6,43 @@
...
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:4
-#, fuzzy, no-wrap
-#| msgid "`--directoryfile` Enable directory to remember registered servers even if the directory is restarted. You'll need to specify a path to a writable file location in which server information is stored.\n"
+#, no-wrap
 msgid "`--directoryfile` Remember registered Servers even if the Directory is restarted. Directory Servers only. See [this guide](Custom-Directories) for further information. \n"
-msgstr "`--directoryfile` Activeer het adresboek om geregistreerde servers te onthouden, zelfs als het adresboek opnieuw wordt gestart. Je moet een pad opgeven naar een beschrijfbare bestandslocatie waarin de serverinformatie wordt opgeslagen.\n"
+msgstr "`--directoryfile` Activeer het adresboek om geregistreerde servers te onthouden, zelfs als het adresboek opnieuw wordt gestart. Je moet een pad opgeven naar een beschrijfbare bestandslocatie waarin de serverinformatie wordt opgeslagen. Zie [deze handleiding](Custom-Directories) voor meer informatie. \n"
```
Comment (pljones) [inline _translator-files/po/nl/Include-Server-Commands.po]: The Dutch has
> `Je moet een pad opgeven naar een beschrijfbare bestandslocatie waarin de serverinformatie wordt opgeslagen.`

but the English does _not_ say
> `You must provide a path to a writable file location where the server information will be stored.`

Technically correct but diversions from the original text become hard to maintain.  Any such additions should be made as changes to the English _first_ and then included in all translations.

File: _translator-files/po/nl/Include-Server-Commands.po [hunk 2]
```diff
@@ -6,47 +6,43 @@
...
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:5
-#, fuzzy, no-wrap
-#| msgid "`-f`  or `--listfilter`     Whitelist servers registering on the server list, format `ip address 1[;ip address 2]` Directory Servers only. See [server types](#server-types)  \n"
+#, no-wrap
 msgid "`-f`  or `--listfilter`     Whitelist Servers registering on the Server list, format `ip address 1[;ip address 2]` Directories only. \n"
-msgstr "`-f`  of `--listfilter`     Whitelist-servers die zich registreren op de serverlijst, formaat `ip-adres 1[;ip-adres 2]` Alleen voor adresboek servers. Zie [servertypen](#servertypen)  \n"
+msgstr "`-f`  of `--listfilter`     Whitelist-servers die zich registreren op de serverlijst, formaat `ip-adres 1[;ip-adres 2]` alleen voor adresboeken. \n"
```
Comment (pljones) [inline _translator-files/po/nl/Include-Server-Commands.po]: You used a different term for white listing elsewhere.  I'd favour consistency -- I think the other place might have been a new translation (can't find it now) -- using whichever is more comfortable.

File: _translator-files/po/nl/Include-Server-Commands.po [hunk 3]
```diff
@@ -135,7 +126,6 @@ msgstr "`-z` of `--startminimized`  Start geminimaliseerd\n"
 
 #. type: Plain text
 #: ../wiki/en/Include-Server-Commands.md:19
-#, fuzzy, no-wrap
-#| msgid "`--serverpublicip`  The public IP address of the server if connecting to a Directory behind the same NAT. See [Notes on Directory Servers](Running-a-Server#points-to-note-about-directories)\n"
+#, no-wrap
 msgid "`--serverpublicip`  The public IP address of the Server if connecting to a Directory behind the same NAT. See [Notes on Directory Servers](Running-a-Server#points-to-note-about-directories)\n"
```
Comment (pljones) [inline _translator-files/po/nl/Include-Server-Commands.po:130]: Uff, English is wrong - @gilgongo, is this on the list?

File: _translator-files/po/nl/Tips-Tricks-More.po [hunk 4]
```diff
@@ -346,3 +323,18 @@ msgid ""
 "<?php loadserverstat ( \"stat1.dat\" ); ?>\n"
 "~~~\n"
 msgstr ""
```
Comment (pljones) [inline _translator-files/po/nl/Tips-Tricks-More.po:325]: Is there no way to mark a block as "no translate required"?

Comment (ann0see) [reply _translator-files/po/nl/Tips-Tricks-More.po:325]: @ignotus666 ?

Comment (henkdegroot) [reply _translator-files/po/nl/Tips-Tricks-More.po:325]: I believe we had this discussion when po4a was introduced and if I recall correct it is not possible.
Current stand is that when you don't add it to the nl text (which I tried in the beginning) it no longer counts for the amount of translation done and often (as these blocks can be large) will get the file below the threshold to get the translation visible.

File: _translator-files/po/nl/Include-Server-Commands.po (hunk 1 above)
Comment (henkdegroot) [reply _translator-files/po/nl/Include-Server-Commands.po]: Looks like I missed a part of the English changes here. Thanks for spotting this!

File: _translator-files/po/nl/Include-Server-Commands.po (hunk 2 above)
Comment (henkdegroot) [reply _translator-files/po/nl/Include-Server-Commands.po]: Will check and update accordingly.

File: _translator-files/po/nl/Tips-Tricks-More.po (hunk 4 above)
Comment (ignotus666) [reply _translator-files/po/nl/Tips-Tricks-More.po:325]: @henkdegroot is correct. It's just ctrl+b in Poedit to copy the source segment over.

**Review by pljones — APPROVED** [review]
Thanks!

**Pull request merged**